### PR TITLE
Move donations to monolith

### PR DESCRIPTION
**Pull request by paked**
Forgive me father, for I have sinned.

This PR adds two new routes:

- GET $FRONTEND/donate
- POST $API/v1/donations

The different tiers of donations are not a thing. I am sorry. Once this gets merge we should make an issue for them.

### CODE REVIEW

**Comment by paked** [comment]
laugh

File: frontend/src/containers/DonationPage/DonationPage.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+      document.head.appendChild(script);
+    } else {
+      onload();
+    }
+  }
+
+  startStripe(e) {
+    this.loadStripe(() => {
+      this.stripeHandler = window.StripeCheckout.configure({
+        key: 'pk_test_yw3YYp18HpZD1j0sR4un7h2N',
```
Comment (MaxWofford) [inline frontend/src/containers/DonationPage/DonationPage.js]: 🥇

File: frontend/src/containers/DonationPage/DonationPage.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+          var data = new FormData()
+
+          data.append("stripe_email", token.email)
+          data.append("stripe_token", token.id)
+          data.append("recurring", this.state.recurring)
+          data.append('amount', this.amountInCents())
+
+          this.setState({status: 'loading'})
+
+          fetch('http://localhost:3000/v1/donations', {method: 'post', body: data })
```
Comment (MaxWofford) [inline frontend/src/containers/DonationPage/DonationPage.js]: 🔢

File: frontend/src/containers/DonationPage/DonationPage.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+            .then(resp => {
+              return resp.json()
+            }).then(data => {
+              if (data.donation_successful) {
+                console.log(data)
```
Comment (MaxWofford) [inline frontend/src/containers/DonationPage/DonationPage.js]: 💯

File: frontend/src/containers/DonationPage/DonationPage.js [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    marginBottom: '20px',
+    marginTop: '10px'
+  }
+}
+
+class Cards extends React.Component {
+  constructor(props) {
+    super(props);
+
+    console.log(props)
```
Comment (MaxWofford) [inline frontend/src/containers/DonationPage/DonationPage.js]: 🎂

**Comment by paked** [comment]
With custom amount:

image

With a standard tier:

image

**Comment by paked** [comment]
Man, we've come a long way since *this* monstrosity:

image

**Comment by MaxWofford** [comment]
Ping me when this is ready for review.

**Comment by paked** [comment]
Just reviewed this with Max in person, and he has given the OK to merge.

**Pull request merged**